app/main.py

```diff
@@ -242,7 +247,7 @@ def saveTranslations(request):
         # TODO Call Kafka Producer Here
         # data to process {audioFile}
 
-        return response.json({"message": "Added One item"})
+        return response.json({"message": "Added One Item"})
```

Comment:
Nit: why capital letters at the beginning of each word?

Reply:
Just a personal preference.